easyrest/views/user_controller.py

```diff
@@ -50,3 +55,38 @@ def sign_up(request):
     except IntegrityError:
         database.rollback()
         raise HTTPForbidden("User already exists!")
+
+
+@view_config(route_name='toggle_activity', renderer='json', request_method='GET')
+@restrict_access(['Administrator', 'Owner', 'Moderator', 'Admin'])
+def toggle_activity(request):
+    """This function is intended to toggle the activity status of a user with a specific id.
+
+    This function processes the route user/toggle_activity/{user_id:\d+}
+    and toggle the activity status of a user depending on the identifier
+    that is extracted from the parameter {user_id}.
+
+    :param request: GET request.
+    :return: If user activity changed successfully:
+                {
+                  "message": "Activity status successfully changed",
+                  "data": [],
+                  "success": true,
+                  "error": null
+                }
+    :raise HTTPNotFound: If user id not found.
+    """
+    log = logging.getLogger(__name__)
+
+    requested_user_id = int(request.matchdict['user_id'])
+    database = request.dbsession
+    requested_user = database.query(User).get(requested_user_id)
```

Comment:
Part of this job done in restrict access (which connected with Token: User must have token, but only Authorized user...)

Reply:
This is because the current user can change the status of another user.